src/components/User/Limits/components/Limits.Manager.vue

```diff
@@ -336,7 +346,15 @@
         }
         return true
       },
-      setFilters () {
+      checkFilters () {
+        if (this.filters.accountRole || this.filters.address) {
+          return true
+        } else {
+          this.$store.dispatch('SET_ERROR', 'Such account does not exist in the system')
+          return false
+        }
+      },
```

Comment:
introduce `selectedTabName` data property to change active tab to avoid such confusing validators

Reply:
Updated